.github/workflows/test.yml

```diff
@@ -84,7 +84,7 @@ jobs:
           node-version: 18.15.0
           cache: yarn
       - name: Build image Phlare
-        run: make docker-image/phlare/build "BUILDX_ARGS=--cache-from=type=gha --cache-to=type=gha"
+        run: make docker-image/pyroscope/build "BUILDX_ARGS=--cache-from=type=gha --cache-to=type=gha"
```

Comment:
`.goreleaser` needs some fixes as well but I'm not sure if it is supposed to be part of this step. Same for helm chart

Reply:
Will try to get weekly builds going next, there is some more work there yes

Reply:
Most recent commit fixed `make deploy`, we should not touch the helm chart for now